pkg/volume/gcepd/attacher.go

```diff
@@ -393,10 +402,42 @@ func (detacher *gcePersistentDiskDetacher) UnmountDevice(deviceMountPath string)
 		if err != nil {
 			return err
 		}
+		target, err := getPathTarget(deviceMountPath)
+		if err != nil {
+			return err
+		}
+		if err = dismount(target); err != nil {
```

Comment:
is there a reason why we only do this for gce pd plugin and not do it as a normal part of Unmount?

Reply:
dismount here should be only performed as part of unmountDevice before detaching. If multiple pods mount the same disk on the same node, dismount will break other pods accessing the disk.

Reply:
Is this specific to pd, or can other block storage drivers use it like ebs, azure block?

Reply:
I think it is not specific to pd.

Reply:
I think we should follow up and try to get this logic into the mount library because it's not specific to pd.

Reply:
Create bug to track it https://github.com/kubernetes/kubernetes/issues/93978